src/api/app/views/webui2/webui/groups/new.html.haml

```diff
@@ -0,0 +1,19 @@
+:ruby
+  @pagetitle = 'Create Group'
+
+.card.mb-3
+  = render partial: 'webui/configuration/tabs'
+  .card-body
+    %h3= @pagetitle
+    = form_for(Group.new, url: group_create_path, method: :post) do |form|
+      .form-group
+        = form.label(:title)
+        %abbr.text-danger{ title: 'required' } *
+        = form.text_field(:title, class: 'form-control', required: true, autofocus: true)
+      .form-group
+        = label_tag(:members)
+        = text_field_tag('group[members]', @members, id: 'group_members', class: 'form-control tag-input', data: { source: user_autocomplete_path })
```

Comment:
Use dashes in case of html ids and classes: `group-members`

Reply:
Following your suggestion still renders the `input` with the ID `group_members`. I will revert this change since it's confusing to have the view with `group-members` and the JavaScript with `group_members` in when fetching the element with jQuery.

Reply:
> Following your suggestion still renders the `input` with the ID `group_members`.

Not on my machine. May it be that you removed the id completely? Because in that case it uses what `text_field_tag` generates which is 'group_members'. In other words you can drop it, if you stick with the underscore version.

> I will revert this change since it's confusing to have the view with `group-members` and the JavaScript with `group_members`

Well, you would have to update any place where the id is used:smiley:

I don't think there is any style guide that would deviate from using hyphens as delimiter for html classes and ids. Similar like ruby using snake_case and js camelCase.

Reply:
I don't remember what I did last week, but it was clearly wrong. I guess I removed the `ID` and this is why it was still with the underscore. Anyway, I changed it now.